atlp-rwanda/ecommerce-app-champions-fe
# #82 Feature:  Search for a product

# What does this PR do
A user should be able to search for a product

# Description of Task to be completed
As a user, I should be able to use the search component
So that, I can easily retrieve records from both the request and approval table

# How should this be manually tested?
clone the repo using git clone repo's URL
change branch git checkout ft-search-product-cp-82
create .env file and paste variables from .env.example
run npm run dev
navigate to Vendor signup
Signup get a password from your email and use it for login
Get authenticated via two factor auth
You will be redirected to vendor's dashboard and have access on vendor's product
you will also see search box and then type anything you want to search

# Relevant pivotal tracker user story
#82
